BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-129: Create and update routes

- Clean routes files
- Clean employees route file
- Delete unused routes files
- Update admins route file
- Update super admin route file

QA:
How to test?
```
Users to test:

Employee:
Email: flune19@desdev.cn
Password: ioAgbm27rl

Admin:
Email epoley8@cam.ac.uk
Password: ECxjgjJOn9


Super Admin:
Email: oclacep@wikimedia.org
Password: Lvt935BO
```

To test this PR you should log with each user and check that you can access to their specific routes:
As employee you could navigate to the pages through the side bar to: the home page, the timesheets page (when accesing through the add hour button in the projects list the select input with the projects will show the project you selected), and the profile page.

As Admin and Super Admin there is no sidebar implemented yet, so you can be able to access only to the home page.
